docs/description/array_ephys.md

```diff
@@ -55,12 +57,16 @@ Through our interviews and direct collaboration on the precursor projects,
  we identified the common motifs to create the Array Electrophysiology Element with the repository hosted at https://github.com/datajoint/element-array-ephys.
 
 Major features of the Array Electrophysiology Element include:
-+ Pipeline architecture defining:
-    + Probe, electrode configuration compatible with Neuropixels probes and generalizable to other types of probes (e.g. tetrodes) - supporting both chronic and acute probe insertion mode
-    + Probe-insertion, ephys-recordings, LFP extraction, clusterings, curations, sorted units and the associated data (e.g. spikes, waveforms, etc.)
-    + Store/track/manage different curations of the spike sorting results
+
++ Pipeline architecture detailed by:
+
+   + Probe, electrode configuration compatible with Neuropixels probes and generalizable to other types of probes (e.g. tetrodes) - supporting both chronic and acute probe insertion modes
+   + Probe-insertion, ephys-recordings, LFP extraction, clusterings, curations, sorted units and the associated data (e.g. spikes, waveforms, etc.)
+   + Store/track/manage different curations of the spike sorting results - supporting both curated clustering and kilosort triggered clustering (no_curation)
```

Comment:
I think I want to be a little more explicit here about `ephys_mode`, so let's put the schema names in backtics
```suggestion
   + Store/track/manage different curations of the spike sorting results - supporting both curated clustering and kilosort triggered clustering (i.e., `no_curation`)
```